qt/widgets/common/CMakeLists.txt

```diff
@@ -839,29 +839,31 @@ set( TEST_FILES
   test/WorkspacePresenter/WorkspacePresenterTest.h
 )
 
-mtd_add_qt_tests (TARGET_NAME MantidQtWidgetsCommonTest
-  QT_VERSION 4
-  SRC ${TEST_FILES}
-  INCLUDE_DIRS
-    ../../../Framework/TestHelpers/inc
-    ../../../Framework/DataObjects/inc
-    ../../../Framework/Crystal/inc
-  TEST_HELPER_SRCS
-    ../../../Framework/TestHelpers/src/TearDownWorld.cpp
-    ../../../Framework/TestHelpers/src/ComponentCreationHelper.cpp
-    ../../../Framework/TestHelpers/src/DataProcessorTestHelper.cpp
-    ../../../Framework/TestHelpers/src/InstrumentCreationHelper.cpp
-    ../../../Framework/TestHelpers/src/WorkspaceCreationHelper.cpp
-  LINK_LIBS
-    ${TARGET_LIBRARIES}
-    DataObjects
-    ${GMOCK_LIBRARIES}
-    ${GTEST_LIBRARIES}
-  MTD_QT_LINK_LIBS
-    MantidQtWidgetsCommon
-  PARENT_DEPENDENCIES
-    GUITests
-)
+if ( ENABLE_MANTIDPLOT )
+  mtd_add_qt_tests (TARGET_NAME MantidQtWidgetsCommonTest
```

Comment:
Is this `if` required? The `mtd_add_qt_tests` function is suposed to operate in a similar manner to the library target and will auto-disable the Qt4 tests if `ENABLE_MANTIDPLOT=False`.

Reply:
No, its not, thanks. I checked the `mtd_add_qt_<suffix> suffix={tests, library, executable}` and think I have fixed these all now to remove unnecessary if-statments.

Reply:
Thanks :+1: